priv/templates/nimble_template/README.md.mix.eex

```diff
@@ -29,27 +29,38 @@
 - Run all tests:
 
   ```sh
-  mix test 
+  mix test
+  ```
+
+- Run all tests with `watch` option:
```

Comment:
I think we didn't add this addon into the mix variant, but here we update the `READ ME for the mix variant` 🤔

Reply:
yup, now we support the mix variant